style.css

```diff
@@ -0,0 +1,77 @@
+
+.title {
+		font-size: 10px;
+}
+
+body {
+		font-family: arial, sans-serif;
+}
+
+.nav {
+		display: flex;
+		justify-content: flex-end;
+		background: #008aff;
+}
+
+.nav a {
+		color: #fff;
+		padding: 15px 10px;
+		text-decoration: none;
+		text-align: center;
+}
+
+.nav a:hover {
+		background: #5c8db7;
+}
+
+@media all and (max-width: 800px) {
+.nav {
+		justify-content: space-around;
+}
+
+.nav a {
+		flex-grow: 1;
+	}
+}
+
+@media all and (max-width: 500px) {
+.nav {
+		flex-direction: column;
+	}
+}
+
+@media (min-width:320px) {
+.title {
+		font-size: 15px;
+	}
+}
+
+@media (min-width: 481px) {
+.title {
+		font-size: 25px;
+	}
+}
+
+@media (min-width: 641px) {
+.title {
+		font-size: 35px;
+	}
+}
+
+@media (min-width: 961px) {
+.title {
+		font-size: 45px;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Expected indentation of 4 spaces (indentation)](https://app.codacy.com/gh/WebHeroSchool/ad/pullRequest?prid=5498083)